content/en/blog/_posts/get-started-with-kubernetes-using-python.md

```diff
@@ -0,0 +1,217 @@
+﻿---
+layout: blog
+title: Get started with Kubernetes (using Python)
+date: 2019-06-20
+---
+
+**Author**: Jason Haley (Independent Consultant)
+
+So, you know you want to run your application in Kubernetes but don’t know where to start. Or maybe you’re getting started but still don’t know what you don’t know. In this blog you’ll walk through how to containerize an application and get it running in Kubernetes.
+
+This walk-through assumes you are a developer or at least comfortable with the command line (preferably bash shell).
+
+## What we’ll do
+1. Get the code and run the application locally
+2. Create an image and run the application in Docker
+3. Create a deployment and run the application in Kubernetes
+
+## Prerequisites
+* [Docker Desktop with Kubernetes](https://www.docker.com/products/kubernetes) installed (see end of blog for enabling Kubernetes)
```

Comment:
Might be better to just list "A Kubernetes service" here, possibly with a link to the various installation methods here: https://kubernetes.io/docs/setup/.

Reply:
Ok, that makes sense I work that in. One of the ideas I had with this blog was get across the message that if you are already working with Docker,  it is easy to also start learning Kubernetes (instead of Docker Swarm) with the tools you already have.